### PR TITLE
chore: Enable modularization

### LINKED ISSUE #11794: Enable modularization
Create a branch based on `mod-run-migration` that can be merged into the main branch:

- [x] Set default of `workflows.enabled` to `true`
- [x] Disable jobs `E2E Tests` and `Integration Tests`
- [x] Fix failing Mod unit tests
- [x] Disable failing Mono unit tests
- [ ] Fix failing `Hedera Node JRS Panel`

### PR DESCRIPTION
**Pull request by netopyr**
Fixes #11794 

- Updates the default configuration path to `data/config/`  in -order for JRS tests to pick correct config
- Changes the `CLASS_ID` of `MerkleHederaState` to use the `CLASS_ID` of `ServicesState` in order for mod-service load a mono-service state
- Added `hedera-node/data/config/application.properties`  and `hedera-node/data/config/genesis.properties` for JRS tests
- Fixed deleting of a directory during upgrade found in Upgrade testing in `FreezeUpgradeActions`
- Changes limits of entities in `InitialModServiceTokenSchema` and `InitialModServiceContractSchema` that seemed to have resolved issues in Tom's testing

Closes #11794 and #11867

### CODE REVIEW

**Review by jeromy-cannon — APPROVED** [review]

File: hedera-node/data/config/application.properties [hunk 1]
```diff
@@ -0,0 +1,16 @@
+bootstrap.throttleDefsJson.resource=throttles-dev.json
+ledger.id=0x03
+staking.periodMins=1
+scheduling.whitelist=ConsensusSubmitMessage,CryptoTransfer,TokenMint,TokenBurn,CryptoApproveAllowance,CryptoUpdate
+# Uncomment a line below to test HAPI operations via workflows
+hedera.workflows.enabled=true
+#hedera.workflows.enabled=ConsensusCreateTopic,ConsensusUpdateTopic,ConsensusDeleteTopic,ConsensusSubmitMessage,ConsensusGetTopicInfo
```
Comment (thomas-swirlds-labs) [inline hedera-node/data/config/application.properties:7]: Nit: do we need this comment in here any more? Maybe it Is left here just to show examples of how to enable workflows on an individual class basis ?

**Review by thomas-swirlds-labs — APPROVED** [review]
LGTM!

**Review by JeffreyDallas — APPROVED** [review]

File: hedera-node/data/config/application.properties (hunk 1 above)
Comment (netopyr) [reply hedera-node/data/config/application.properties:7]: Good catch. I am not even sure if this still works. This file needs some cleanup. We should do this in a follow-up PR.

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM ! Thanks @netopyr

File: hedera-node/data/config/application.properties (hunk 1 above)
Comment (Neeharika-Sompalli) [reply hedera-node/data/config/application.properties:7]: Agreed! Will change in next PR. Thanks @thomas-swirlds-labs @netopyr

**Review by iwsimon — APPROVED** [review]
LGTM

**Review by david-bakin-sl — APPROVED** [review]
yes, LGTM

**Pull request merged**